airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/packaging.py

```diff
@@ -63,6 +63,24 @@ def _run(self, connector: Connector) -> CheckResult:
         return self.create_check_result(connector=connector, passed=True, message="PyPi publishing is enabled")
 
 
+class CheckManifestOnlyConnectorBaseImage(PackagingCheck):
```

Comment:
@natikgadzhi I think we need to add an instance of this class to the `ENABLED_CHECKS` list at the bottom of this file for it to run?

Reply:
BREH you're right.